ocaml/tests/test_vgpu_type.ml

```diff
@@ -202,6 +137,34 @@ module NvidiaTest = struct
       ]
     end)
 
+  module HostDriverMultipleVgpuSupport = Generic.Make(struct
+      module Io = struct
+        type input_t = (string * string list)
+        type output_t = bool
+
+        let string_of_input_t (host_driver_version, supported_driver_versions) =
+          Printf.sprintf "(%s, %s)" host_driver_version (String.concat "," supported_driver_versions)
+
+        let string_of_output_t x = string_of_bool x
+      end
+
+      let transform (host_driver_version, supported_versions) = Xapi_vgpu_type.Vendor_nvidia.is_host_driver_support_multi_vgpu ~host_driver_version ~supported_versions
+
+      let tests = [
+        ("430.19",["430.19"]), true; (* Test specific version should match *)
+        ("430.19",["430.89";"430.19"]), true; (* Test specific version should match *)
+        ("430.19",["430.19";"typo_error"]), true; (* Test specific version should match *)
+        ("430.19",["430.18"]), false; (* Test specific version should not match *)
+        ("430.18",["430.19"]), false; (* Test specific version should not match *)
+        ("430.400",["430.200+"]), true; (* LCM release, higher version should match *)
+        ("430.100",["430.200+"]), false; (* LCM release, lower version should not match *)
+        ("530.400",["430+.200"]), true; (* master release, higher version should match *)
+        ("100.400",["430+.200"]), false; (* master release, lower version should not match *)
+        ("530.400",["430+.200";"typo_error"]), true; (* LCM release*)
+        ("530.400",["430+.200+"]), true; (* LCM release*)
```

Comment:
How about `("100.100",["430.200+"]), false`?

Reply:
I will add this one